### DISCUSSION TITLE
Group coordinate system

### DISCUSSION THREAD

**Discussion by w934423231** [fabricjs/fabric.js#8005] [Q&A]
I use group to wrap two paths. It seems that there is an exception when the position triggered by the mouse changes to the local position of the element. It may be my problem, but I really can't find the problem

**Comment by w934423231** [comment]
https://codepen.io/w934423231/pen/gOvZEeO?editors=0010

This is a short case of mine. It is the same as my code, but the results are a little different

**Comment by ShaMan123** [comment]
It is design. You MUST read the docs.
There are many duplicate discussion and issues with the group label.

**Reply by w934423231** [reply]
How can I implement this function? How can I combine the two paths without a group

**Reply by w934423231** [reply]
I have been searching the Internet for a long time, but I haven't found a good solution. I am worried about this problem

**Reply by ShaMan123** [reply]
No need for worrying.
You haven't searched fabric good enough.
`fabric.util.sendPointToPlane`
It is solved in v6.

**Reply by w934423231** [reply]
Has this V6 been updated now

**Reply by w934423231** [reply]
Where can I find this document

**Reply by ShaMan123** [reply]
search the repo.

**Reply by ShaMan123** [reply]
https://github.com/fabricjs/fabric.js/search?q=sendPointToPlane

**Reply by w934423231** [reply]
OK, thank you. Let me see if this problem can be solved

**Reply by w934423231** [reply]
Hello, I would like to ask you what you have changed. I will package your code and replace it. Many functions of my project are different from the original

**Reply by w934423231** [reply]
The previously set custom control point is no longer available

**Reply by w934423231** [reply]
How should I be compatible with these things

**Comment by pjy1711040554** [comment]
look look   https://codepen.io/pjy1711040554/pen/rNdabMQ
